packages/lab/src/calendar/internal/CalendarNavigation.tsx

```diff
@@ -112,22 +104,34 @@ function useCalendarNavigation() {
     canNavigatePrevious,
     selectedMonth,
     selectedYear,
+    isOutsideAllowedMonths,
   };
 }
 
-const ListItemWithTooltip: ListItemType<DropdownItem> = ({
-  item,
-  label,
-  ...props
-}) => (
-  <Tooltip
-    placement="right"
-    disabled={!item?.disabled}
-    content="This month is out of range"
-  >
-    <ListItem {...props}>{label}</ListItem>
-  </Tooltip>
-);
+const OptionWithTooltip = ({
+  value,
+  children,
+  disabled,
+  tooltipContent,
+}: OptionWithTooltipProps) => {
+  const { activeState, openState } = useListControlContext();
+  const open = activeState?.value === value ? true : undefined;
```

Comment:
Worth checking but I believe this can be fixed now:

```suggestion
  const open = activeState?.value === value;
```